protocols/loader.py

```
@@ -36,17 +32,18 @@ def task():
     result = {{'success': bool(randint(0, 1)), 'action': 'default_execution'}}
     return result
 '''
-        with open(protocol_file, 'w') as f:
+        with open(protocol_file, "w") as f:
```

Pylint (3.3.7)

[warning] Using open without explicitly specifying an encoding

(W1514)